serial/serialposix.py

```diff
@@ -345,10 +348,7 @@ def open(self):
             self._reconfigure_port(force_update=True)
 
             try:
-                if not self._dsrdtr:
-                    self._update_dtr_state()
-                if not self._rtscts:
-                    self._update_rts_state()
+                self._update_rts_and_dtr_state()
```

Comment:
I think I need to look at this a bit more. The change here would allow the rts and dtr io lines to be set even when in one of the flow control modes. That is probably wrong.